### PR TITLE
Edited Help Desk process based on conversation with Fernando and Adam. Ready to merge.

### PR DESCRIPTION
**Pull request by nlharris**

### CODE REVIEW

File: Help_Desk_process.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon
-Unless the Help Desk managers can answer the question themselves, they assign the issue to the relevant box lead.
-The box lead assigns external issues within two days to someone who can address the issue.
+Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon.
+Unless the Help Desk managers can answer the question themselves, they assign the issue to the person they think is most likely to be able to address it.
 
-Within three business days, the assignee looks at the assigned issue and triages it, entering a comment in the ticket to indicate:
+**Within two business days, the assignee needs to look at the assigned issue and enter a comment in the ticket to indicate whether they are the right assignee**. If they are not, then they should indicate in a comment that they can't address it, and they should reassign to the right person if they know who that is, or say that they don't know who it should be assigned to.
```
Comment (fperez) [inline Help_Desk_process.md]: I would rephrase a bit for readability as

Within two business days, the assignee needs to comment to either:
- acknowledge that they will handle the issue (even if not immediately), or
- indicate that they are not the right person to do so.  In this case, They should reassign to the right person if they know who that is, or request that it be reassign if they don't.

Comment (nlharris) [inline Help_Desk_process.md]: Sure, that sounds good. Thanks.

File: Help_Desk_process.md [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
 
-  a. The answer is... (Enter text to be forwarded to the user by Help Desk personnel.)(The assignee can then resolve (not close) the ticket.)
-  
-  b. I will need to do more research to answer this. I will answer within X days.
+When you respond to a ticket that you are assigned, it's important to add some indication of how hard you think the issue is likely to be to fix, and when you will have time to work on it. Please also let us know if this is blocked by another issue--another task must be completed (by you or by someone else) before this can be fixed. (If possible, include link(s) to other JIRA tickets.)
 
-    (Also, indicate if this is a question that is likely to come up again and thus should be added by UE to the relevant documentation.)
-    
-2. This is a task.
+It's ok if you don't have time to work on the issue right away--the important thing is to let the EC know your time frame so that if the issue is important, we can reassign resources appropriately.
```
Comment (fperez) [inline Help_Desk_process.md]: This is the first time `EC` is used, define.

File: Help_Desk_process.md (hunk 1 above)
Comment (nlharris) [inline Help_Desk_process.md]: Should I edit it in this PR, or were you going to?

File: Help_Desk_process.md (hunk 2 above)
Comment (nlharris) [inline Help_Desk_process.md]: Ok, will do

File: Help_Desk_process.md (hunk 1 above)
Comment (nlharris) [inline Help_Desk_process.md]: I will make the change.

Comment (fperez) [inline Help_Desk_process.md]: Go ahead, I'm just commenting for now, I'll let you make the edits.

File: Help_Desk_process.md [hunk 3]
```diff
@@ -1,44 +1,46 @@
...
+4. This is a request for an improvement or new feature.<br/>
+  a. This was easy to do so I did it. (Once the fix is committed, merged and deployed, the assignee can close the ticket.)<br/>
+  b. It is on our roadmap and will likely be included in release X.<br/>
+  c. It's not on our roadmap, but it's a good idea, so EC should consider adding it to our list.<br/>
   d. It's not on our roadmap and shouldn't be.
 
-UE will send the EC a monthly report about JIRA performance.
+An important part of the JIRA workflow is to return to the ticket when the issue is fixed, add an appropriate comment, and (if you think the issue is completely fixed and deployed, and the reporter should be able to see the new behavior) close the ticket.
+
+The UE will send the EC a monthly report about JIRA performance, including the identification of top issues and bottlenecks.
```
Comment (fperez) [inline Help_Desk_process.md]: Define UE

File: Help_Desk_process.md [hunk 4]
```diff
@@ -1,44 +1,46 @@
...
-  c. It's not on our roadmap, but it's a good idea, so EC should consider adding it to our list.
-  
+4. This is a request for an improvement or new feature.<br/>
+  a. This was easy to do so I did it. (Once the fix is committed, merged and deployed, the assignee can close the ticket.)<br/>
+  b. It is on our roadmap and will likely be included in release X.<br/>
+  c. It's not on our roadmap, but it's a good idea, so EC should consider adding it to our list.<br/>
   d. It's not on our roadmap and shouldn't be.
 
-UE will send the EC a monthly report about JIRA performance.
+An important part of the JIRA workflow is to return to the ticket when the issue is fixed, add an appropriate comment, and (if you think the issue is completely fixed and deployed, and the reporter should be able to see the new behavior) close the ticket.
```
Comment (fperez) [inline Help_Desk_process.md]: I would replace this whole paragraph just by "Don't forget to close your tickets when done!". Too wordy...

**Comment by fperez** [comment]
Looks great, small fixes, once updated good to merge, thanks!!

**Comment by fperez** [comment]
Will wait till others in the project have a chance to comment further before merging, I'm done with my review and happy with this. Great work.

**Comment by kkellerlbl** [comment]
Once a POC rotation is established, I'd like to figure out whether/how we should include information on that in this document (or perhaps just link to the SLA docs from here?).

**Comment by nlharris** [comment]
@kkellerlbl, I'm not sure what you mean--do you mean so that systems issues can be assigned to the relevant POC?

**Comment by kkellerlbl** [comment]
I think that's part of what I mean, but honestly I'm not 100% sure of all of what I mean.  I need to let it stew in my head for a little bit, but I was hoping that it could stew in other people's heads too.

**Comment by dangunter** [comment]
So yeah people like Matt and Bill need a type of SLA for % time doing bug
fixes.

(sent from my phone)

**Comment by kkellerlbl** [comment]
<blockquote>So yeah people like Matt and Bill need a type of SLA for % time doing bug fixes.</blockquote>

I'd agree but I don't know exactly what that would look like (and is probably another document, or could be part of the SLA document, but not something to be included here).

**Comment by nlharris** [comment]
> So yeah people like Matt and Bill need a type of SLA for % time doing bug fixes.

I've been advocating something like that for years.

But I agree with Keith that that's outside the scope of this doc.

**Comment by dangunter** [comment]
ok, then maybe we need a "user support" doc.

**Comment by fperez** [comment]
This one seems to be settling, with the notion that a potential 'user support doc' can be expanded further. Keep in mind folks, merging a PR doesn't mean that we never touch this document, so further edits are always possible...

But this seems to be a reasonable starting point, so let's move ahead...
